### PR TITLE
Module refactor

### PR DESCRIPTION
**Pull request by CLRM1**
In this pull request, we have completed the following. 
1. Added a module. 
2. Added a README with a blog post and project description. 
3. Refactored the sales analyst class and removed a duplicate method. 
4. Removed extra white space.

### CODE REVIEW

**Comment by amsalmeron** [comment]
WOOOOHOOOOOO!!

File: lib/invoice.rb [hunk 1]
```diff
@@ -8,23 +8,21 @@
...
-                    else
-                      Time.parse(attributes[:created_at])
-                    end
-    @updated_at = if attributes[:updated_at].class == Time
-                    attributes[:updated_at]
-                  else
-                    Time.parse(attributes[:updated_at])
-                  end
+    @created_at = attributes[:created_at].class == Time ? attributes[:created_at] : Time.parse(attributes[:created_at])
+    @updated_at = attributes[:updated_at].class == Time ? attributes[:updated_at] : Time.parse(attributes[:updated_at])
```
Comment (alexGrandolph) [inline lib/invoice.rb]: thanks hound!

Comment (amsalmeron) [reply lib/invoice.rb]: Lets try to make the lines within 80 char.

**Comment by James-Harkins** [comment]
Excellent work yall!

File: lib/rowable.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module Rowable
+
+  def rows
+    rows = CSV.read(@filename, headers: true, header_converters: :symbol)
```
Comment (amsalmeron) [inline lib/rowable.rb]: We don't need to assign rows, maybe we take the assignment operator and variable name out?

**Comment by alexGrandolph** [comment]
cowabunga!!!!!!!!! great job on this project yall

File: lib/sales_analyst.rb [hunk 3]
```diff
@@ -246,22 +240,12 @@ def merchants_with_pending_invoices
...
-        end
-      end.compact.sum
-    end
-    month_merchant_items.find_all {|merchant, item_count| item_count == 1}.flatten
+    month_index = Time.parse(month).month
+    one_item_merchants = {}
+    merchants_with_only_one_item.each {|merchant|
+      one_item_merchants[merchant] = merchant.created_at.month}
+    one_item_merchants.map {|merchant, created_at|
+      merchant if created_at == month_index}.compact
```
[the remark this answers is not in the document]
Comment (alexGrandolph) [reply lib/sales_analyst.rb:248]: ????

File: lib/rowable.rb (hunk 2 above)
Comment (alexGrandolph) [reply lib/rowable.rb]: lets fix this!

**Pull request merged**